components/cam/src/control/scamMod.F90

```diff
@@ -678,13 +678,13 @@ subroutine readiopdata(iop_update_surface,hyam,hybm)
        'readiopdata.F90', __LINE__)
 
 !
-!     if the dataset is a CAM generated dataset set use_camiop to true
-!       CAM IOP datasets have a global attribute called CAM_GENERATED_IOP      
+!     if the dataset is a CAM generated dataset set use_replay to true
+!       E3SM IOP datasets have a global attribute called E3SM_GENERATED_IOP      
```

Comment:
line 682 global attribute "E3SM_GENERATED_FORCING" instead of "E3SM_GENERATED_IOP"